### DISCUSSION TITLE
Substitute: Plotly Plugin

### DISCUSSION THREAD

**Discussion by PriyanshBordia** [grafana/grafana#52803] [Plugins]
Q. Is there any substitute for the `Plotly` plugin in version > `8.x`. I want to plot the curve with an x-axis as a query column.

**Comment by leeoniya** [comment]
we have an XY Chart alpha panel that should officially land by v10. for now you can try it out by enabling alpha in ypur config.

https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/#enable_alpha-1

**Reply by PriyanshBordia** [reply]
Hi @leeoniya, thank you for the reply. I have tried this panel, but somehow this does not seem to work. No data is plotted.

**Reply by leeoniya** [reply]
please open an issue with your dataset and what you're expecting to see. it's difficult to help when you just say "it doesnt work" 🤷

**Reply by PriyanshBordia** [reply]
Ok. Will do that

**Comment by mikhail-vl** [comment]
@PriyanshBordia You can try Apache ECharts plugin https://github.com/volkovlabs/volkovlabs-echarts-panel.

**Reply by PriyanshBordia** [reply]
Hi @mikhail-vl, thank you for the reply. I have checked them, but they did not work well.

**Reply by mikhail-vl** [reply]
@PriyanshBordia, you are probably talking about the outdated version. We forked it recently and updated to the latest version of Apache ECharts and Grafana 9.0.2.

**Comment by tolzhabayev** [comment]
You can also try out https://grafana.com/grafana/plugins/nline-plotlyjs-panel/

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
